### ISSUE TITLE
Generated docstring does not include `:rtype` when return type annotation is present

### ISSUE THREAD

**Issue by debonte** [microsoft/pylance-release#6470]
## Environment data
-   Pylance version: 2024.9.102
-   OS and version: Windows 11
-   Python version: 3.9.12

## Code Snippet

```python
def foo(param1) -> int:
    """"""
    return 1
```

## Repro Steps

1. Trigger completion within `""""""`

## Expected behavior

`:rtype` is included because the return type (`int`) is specified.

## Actual behavior

```python
    """Docstring for foo
    
    :param param1: Description
    :type param1:  """
```

**Comment by KacieKK** [comment]
This issue has been fixed in prerelease version 2024.11.101, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202411101-18-November-2024-PreRelease)

**Comment by Judopark** [comment]
> ## Environment data
> * Pylance version: 2024.9.102
> * OS and version: Windows 11
> * Python version: 3.9.12
> 
> ## Code Snippet
> def foo(param1) -> int:
>     """"""
>     return 1
> ## Repro Steps
> 1. Trigger completion within `""""""`
> 
> ## Expected behavior
> `:rtype` is included because the return type (`int`) is specified.
> 
> ## Actual behavior
>     """Docstring for foo
>     
>     :param param1: Description
>     :type param1:  """

def foo(param1) -> int:
    """"""
    return 1
